index.js

```diff
@@ -15,12 +15,17 @@ module.exports = function (base, country, language, callback) {
         var out = {};
         async.eachSeries(files, function (ent, next) {
             findAllBundlesFromRoot(ent, function (err, o) {
-                out[path.relative(base, ent).replace(/(.*)\/(.*)/, "$2-$1")] = o;
+                out[getKey(base, ent)] = o;
```

Comment:
Oh that is tidy! I'd have inlined it but either way works. Is there a better name for it than 'getKey'? Perhaps 'localeFromPath' ?